### DISCUSSION TITLE
Small adjustment to azure upload

### DISCUSSION THREAD

**Discussion by sak-improsec** [EricZimmerman/KapeFiles#793] [Ideas]
Hi Eric,
My team and I have been using KAPE for almost a year, and have built our forensic platform around it. 

One thing I would like to request is the ability for the --asu flag to drop the target and module output in the same folder in Azure, rather than one for target and one for module:

![image](https://user-images.githubusercontent.com/99182650/235453011-128f5370-3195-493e-b59f-1bb2959e24ed.png)

Usecase: Full triage image uploaded to azure, along with live queries from modules like netstat, pslist, ip-config, and so on, and not having to build logic around which live data is correlated with a specific triage image.

My current "solution" is to "sneak" the module output in with the target by setting --mdest to --tdest/moduleoutput/ and as KAPE zips everything in --tdest, everything will be included. But im not a fan of this hacked way of doing it.

**Comment by AndrewRathbun** [comment]
Maybe this is best brought up as an issue and not a discussion?

**Comment by sak-improsec** [comment]
I was under the impression that it might be intended behaviour. I will re-post as an issue :)
